AutoCollection/ClientRequests.ts

```diff
@@ -20,6 +20,8 @@ class AutoCollectClientRequests {
 
     public static INSTANCE: AutoCollectClientRequests;
 
+    private static requestNumber = 1;
```

Comment:
The parentId is a hierarchical Id of the form `|root.suffix.` or `|freshRoot.`, and `suffix` and `freshRoot` are randomly generated for this particular request. Assuming that `parentId` is unique for this particular incoming request (which is reasonable, a collision of suffix or newly created GUID is unlikely) the `uniqueParentId` will be unique for each outgoing request unless `requestNumber` is ever reset. If the node server restarts or there are multiple servers, they should all be generating different `parentId`s